students/JaeKim/session04/mailroom.py

```diff
@@ -0,0 +1,91 @@
+import tempfile
+
+#!/usr/bin/env python3
+
+"""
+mailroom assignment
+"""
+
+players = {
+    "LeBron James": [1000000, 2000000, 3000000],
+    "Dwyane Wade": [100000, 20000, 30000],
+    "Carmelo Anthony": [1],
+    "Kevin Durant": [1, 2],
+    "Michael Jordan": [2]
+}
+
+
+def thank_you(name):
+    if name.lower() == "list":
+        for player in players:
+            print(player, players[player])
+
+        print("\n")
+    elif name not in players:
+        print("Enter a donation for " + name)
+        donation = int(input(" => "))
+        players[name] = [donation]
+        print("Hi {},\n\n"
+              "Thank you for your donation of {} to the research center.\n\n"
+              "Have a great day!".format(name, donation))
+    elif name in players:
+        donation = sum(players[name])
+
+        email = ("Hi {},\n\n"
+              "Thank you for your donation of {} to the research center.\n\n"
+              "Have a great day!".format(name, donation))
+
+        print(email)
+
+
+def send_all_donors():
+    for player in players:
+        donation_sum = sum(players[player])
+        email_path = tempfile.gettempdir() + '\\' + player + ".txt"
```

Comment:
This line not working for me on Mac OS.  In later versions of mailroom you may want to put in an option for non-Windows users.